### PR TITLE
Lab 2 notes

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: glossary.html [hunk 1]
```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="Malcolm Larmond" content="CSD120 Class Work" />
+    <meta name="author" content="Malcolm Larmond" />
+    <meta name="description" content="CSD120 website">
```
Comment (rmartin-sc) [inline glossary.html:7]: author and description are separate meta elements

File: glossary.html [hunk 2]
```diff
@@ -38,12 +39,16 @@
...
+          <dd>...</dd>
+          <dt>Code Editors</dt>
+          <dd>...</dd>
+          <dt>URL Links</dt>
+          <dd>...</dd>
+          <dt>Git Hub</dt>
+          <dd>...</dd>
+          <dt>Island Life</dt>
+          <dd>...</dd>
         </dl>
```
Comment (rmartin-sc) [inline glossary.html:52]: In a definition list there should be pairs of `<dt>` and `<dd>` elements

File: glossary.html [hunk 3]
```diff
@@ -52,28 +57,19 @@ <h1>Glossary</h1>
         <blockquote
           cite="https://www.quotetab.com/quote/by-bill-laswell/computers-and-electronic-music-are-not-the-opposite-of-the-warm-human-music-its?source=electronic-music"
         >
-          <q
-            >Computers and electronic music are not the opposite of the warm
-            human music. It's exactly the same.</q
-          >
+          Computers and electronic music are not the opposite of the warm
+            human music. It's exactly the same.
```
Comment (rmartin-sc) [inline glossary.html:61]: No need for a `<q>` element inside a `<blockquote>`.  `<q>` is the inline version of `<blockquote>` which is a block-level element

File: glossary.html [hunk 4]
```diff
@@ -52,28 +57,19 @@ <h1>Glossary</h1>
...
 
-          <div class="span">
-            <author> &#8212; Bill Laswell </author>
-          </div>
+            <cite>&#8212; <span class="author">Bill Laswell</span></cite>
```
Comment (rmartin-sc) [inline glossary.html:63]: Here's how this author line should look, using `<cite>` and `<span>`

File: glossary.html [hunk 5]
```diff
@@ -52,28 +57,19 @@ <h1>Glossary</h1>
...
         </blockquote>
-        <br /><br /><br />
 
         <p>
-          &#169; 2019 <a href="mailto: radclash@gmail.com">Malcolm Larmond</a>
+          &#169; 2020 <a href="mailto:radclash@gmail.com?subject=Message%20from%20csd120%20website">Malcolm Larmond</a>
```
Comment (rmartin-sc) [inline glossary.html:67]: Missing subject

File: glossary.html [hunk 6]
```diff
@@ -52,28 +57,19 @@ <h1>Glossary</h1>
...
         </p>
       </footer>
       <!--Found Me-->
 
-      <a href="#top">◤ Top</a>
+      <a href="#top" title="Back to top">◤ Top</a>
```
Comment (rmartin-sc) [inline glossary.html:72]: Missing title

File: glossary.html [hunk 7]
```diff
@@ -52,28 +57,19 @@ <h1>Glossary</h1>
...
     </div>
   </body>
-</html>
-
-<a>
-  <br />
-</a>
+</html>
```
Comment (rmartin-sc) [inline glossary.html:75]: Stray HTML outside of `<html>` element (don't forget to validate your code!)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+      <p>
+        My goal is too understand as much as I can and find gainful emplowment
+        related to computers. I have some family connections that may help me
+        get an intership at a local business. I am also willing to travel
+        anywehere in Canada to work. Perhaps Vancouver, Toronto, or Montreal.
+        That being said, a lot of work is becoming remote in the computer
+        realm so I could also remain here in Sault Ste Marie.
+      </p>
+
+      <h2>My Favourite Things</h2>
```
Comment (rmartin-sc) [inline index.html:60]: You had this as `<h3>`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+      </ol>
+
+      <h3>Haiku - Pot Holes</h3>
+      <p>
+        Meandering thoughts
+        <br>
+        suddenly see pot holes turn 
+        <br>
+        into the lake<br>
+        <cite>by <span class="author">Malcolm L.</span></cite>
```
Comment (rmartin-sc) [inline index.html:80]: The author line should look like this with `<cite>` and `<span>` elements

**Review by MalcolmXL — APPROVED** [review]

**Pull request merged**